java/daikon/dcomp/DCRuntime.java

```diff
@@ -1135,7 +1148,8 @@ public static void enter(Object[] tag_frame, @Nullable Object obj, int mi_index,
       Throwable stack = new Throwable("enter");
       StackTraceElement[] ste_arr = stack.getStackTrace();
       StackTraceElement ste = ste_arr[1];
-      if (ignore_toString && ste.getMethodName().equals("toString")) {
+      // Simplifies printouts for debugging if we ignore toString.
+      if (ste.getMethodName().equals("toString")) {
```

Comment:
Does this logic change affect behavior?  When `debug` is true and the method is `toString`, the method returns early, not executing any of the code after the `if (debug)` conditional.  That seems wrong to me; am I missing something?

The same comment applies to the hunk below.

Reply:
I ran an experiment and disabled the check.  Did not seem to make any difference in the debug output so I will remove the code.